### PR TITLE
Implement loading animations and notifications at specific values

### PR DESCRIPTION
**Pull request by ChanggeonLee**
Implemented to perform loading animation while connected to MQTT server
Implemented to blink red and blue according to the set MAX and MIN

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
         "build": "webpack"
     },
     "dependencies": {
+        "@material-ui/core": "^4.11.0",
```
Comment (soaple) [inline package.json]: Material-UI is too big package.
So, how about using the package below, if you just intend to use progress.
[https://github.com/davidhu2000/react-spinners](url)

**Review by soaple — CHANGES_REQUESTED** [review]
Please check the comments.

**Comment by ChanggeonLee** [comment]
Change package from material-ui to react-spinners

**Review by soaple — APPROVED** [review]
LGTM (Looks Good To Me)

**Pull request merged**